doc/release-notes/0.38/0.38.md

```diff
@@ -63,9 +63,19 @@ The following table covers notable changes in v0.38.0. Further information about
 <td valign="top">OpenJ9 no longer tests OpenJDK 11 on CentOS 6.10 after 0.36.1 release and might stop testing OpenJDK 8 in the future.</td>
 </tr>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16974">#16974</a></td>
+<td valign="top">New <tt>-XX:[+|-]HandleSIGUSR2</tt> option is added to control the handling of the <tt>SIGUSR2</tt> signal by the VM signal handler.</td>
+<td valign="top">All versions (All operating systems except Windows)</td>
+<td valign="top">In the absence of signal chaining, the VM does not allow application signal handlers for certain signals that are used internally by the VM, including the <tt>SIGUSR2</tt> signal. You can use the <tt>-XX:-HandleSIGUSR2</tt> option instead, whereby the VM signal handler is not installed on VM startup. Therefore, the application signal handler, if available, takes over the handling of the <tt>SIGUSR2</tt> signal.</td>
+</tr>
 
-
-
+<tr>
+<td valign="top"><a href="https://github.com/orgs/eclipse-openj9/projects/1">Project 1</a></td>
+<td valign="top">Checkpoint/Restore In Userspace (CRIU) support preview is available.</td>
+<td valign="top">OpenJDK 11 and later (Linux&reg; x86, Linux on POWER&reg; (Little Endian), and Linux on IBM Z&reg;)</td>
```

Comment:
I don't think we want to claim support for zLinux, @tajila ?

Reply:
Its enabled on zlinux. Since we are describing it has a preview feature I dont think its an issue to ad zlinux to the list